app/javascript/src/NewService/components/FormTemplate.jsx

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import React from 'react'
+import {CSRFToken} from 'utilities/utils'
+import type {ServiceDiscoveryFormTemplateProps, FormTemplateProps} from 'NewService/types'
+
+const getServiceDiscoveryProps = (): ServiceDiscoveryFormTemplateProps => {
+  return {
+    hiddenServiceDiscoveryInput: {
+      value: 'discover',
+      name: 'service[source]',
+      id: 'service_source'
+    },
+    serviceDiscoverySelect: {
+      namespace: {
+        htmlFor: 'namespace',
+        label: 'Namespace',
```

Comment:
I remember of an old request to change this to "Project" in order to make it more OpenShift-like.

WDTY, @thomasmaas ?

Reply:
Maybe but as this is a pr about refactoring I think we should not change it here as that would require docs update.